ui/App.svelte

```diff
@@ -14,7 +14,7 @@
   import IdentityCreation from "./Screen/IdentityCreation.svelte";
   import DesignSystemGuide from "./Screen/DesignSystemGuide.svelte";
   import Help from "./Screen/Help.svelte";
-  import Network from "./Screen/Network.svelte";
+  import Settings from "./Screen/Settings.svelte";
```

Comment:
Should be moved down after the `Search` import.

Reply:
👍